phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerWineToolsTab.java

```diff
@@ -54,29 +86,17 @@ private void populate() {
         toolsContentPane.setPrefColumns(3);
         toolsContentPane.getStyleClass().add("grid");
 
-        Button configureWine = new Button(tr("Configure Wine"));
-        configureWine.getStyleClass().addAll("toolButton", "configureWine");
-        configureWine.setOnMouseClicked(event -> {
-            this.lockAll();
-            winePrefixContainerController.runTool(container, "ConfigureWine", this::unlockAll,
-                    e -> Platform.runLater(() -> new ErrorMessage("Error", e).show()));
-        });
-
-        Button registryEditor = new Button(tr("Registry Editor"));
-        registryEditor.getStyleClass().addAll("toolButton", "registryEditor");
-        registryEditor.setOnMouseClicked(event -> {
-            this.lockAll();
-            winePrefixContainerController.runTool(container, "WineRegistryEditor", this::unlockAll,
-                    e -> Platform.runLater(() -> new ErrorMessage("Error", e).show()));
-        });
-
-        Button rebootWindows = new Button(tr("Windows reboot"));
-        rebootWindows.getStyleClass().addAll("toolButton", "rebootWindows");
-        rebootWindows.setOnMouseClicked(event -> {
-            this.lockAll();
-            winePrefixContainerController.runInPrefix(container, "RebootWine", this::unlockAll,
-                    e -> Platform.runLater(() -> new ErrorMessage("Error", e).show()));
-        });
+        for (EngineToolDTO tool : tools) {
```

Comment:
Why not use `MappedList` and `Bindings#bindContent`?

Reply:
Not sure how you would like to have it. Can we merge it and you can do the change afterwards?

Reply:
Basically you could use:

```java
MappedList<...> toolButtons = new MappedList<>(FXCollections.observableArrayList(tools), tool -> {
   ...
   return toolButton; 
});
```
Afterwards you can use 
```java
Bindings.bindContent(toolsContentPane.getChildren(), toolButtons);
```

Reply:
Ok. What's the advantage?

Reply:
It's shorter and it fits better to the way our tabs are created, right?

Reply:
Shorter I don't know. But the second point is true.